databricks/koalas/groupby.py

```diff
@@ -2034,12 +2083,16 @@ def nunique(self, dropna=True):
         Name: value1, dtype: int64
         """
         if dropna:
-            stat_function = lambda col: F.countDistinct(col)
+
+            def stat_function(col):
+                return F.countDistinct(col)
```

Comment:
nit: I think we can leave this as it was before. Is there any special reason in you mind changing this ??